x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/apikey/AbstractCreateApiKeyRequest.java

```diff
@@ -94,6 +98,9 @@ public ActionRequestValidationException validate() {
                 validationException
             );
         }
+        if (refreshPolicy == null) {
+            validationException = addValidationError("refresh policy is required", validationException);
+        }
```

Comment:
The transport request requires the refresh policy to be set, and there is no default.
The values are set in the REST handlers.